src/Service/TaskHandler/SuggestReviewerHandler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service\TaskHandler;
+
+use App\Api\Issue\IssueApi;
+use App\Api\PullRequest\PullRequestApi;
+use App\Api\Status\Status;
+use App\Api\Status\StatusApi;
+use App\Command\SuggestReviewerCommand;
+use App\Entity\Task;
+use App\Service\RepositoryProvider;
+use App\Service\TaskScheduler;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class SuggestReviewerHandler implements TaskHandlerInterface
+{
+    private $issueApi;
+    private $repositoryProvider;
+    private $pullRequestApi;
+    private $statusApi;
+    private $scheduler;
+
+    public function __construct(PullRequestApi $pullRequestApi, IssueApi $issueApi, RepositoryProvider $repositoryProvider, StatusApi $statusApi, TaskScheduler $scheduler)
+    {
+        $this->issueApi = $issueApi;
+        $this->repositoryProvider = $repositoryProvider;
+        $this->pullRequestApi = $pullRequestApi;
+        $this->statusApi = $statusApi;
+        $this->scheduler = $scheduler;
+    }
+
+    /**
+     * We want to check if the PR has no activity, then suggest a reviewer.
+     */
+    public function handle(Task $task): void
+    {
+        $repository = $this->repositoryProvider->getRepository($task->getRepositoryFullName());
+        if ($this->issueApi->hasActivity($repository, $task->getNumber())) {
+            return;
+        }
+
+        $status = $this->statusApi->getIssueStatus($task->getNumber(), $repository);
+        if (Status::NEEDS_REVIEW === $status || null === $status) {
+            $this->pullRequestApi->findReviewer($repository, $task->getNumber(), SuggestReviewerCommand::TYPE_SUGGEST);
+        } else {
+            // Try again later
+            $this->scheduler->runLater($repository, $task->getNumber(), Task::ACTION_SUGGEST_REVIEWER, new \DateTimeImmutable('+20hours'));
+        }
```

Comment:
@ogizanagi I've added this logic. 

If the PR does not have status "needs review" we will try again 20 hours later. Just until somebody else has made a comment.

Reply:
👍  LGTM